lib/diagnostic.js

```diff
@@ -31,16 +34,26 @@ const sum = function (previous, current) {
 // you may use any feature of JavaScript
 // to solve this problem
 const hasNone = function (array, callback) {
-
-}
+  array.forEach((e, i, a) => {
```

Comment:
So this would actually return `true` or `false` for each element. What you want is a function that will return EITHER `true` or `false`